index.js

```diff
@@ -33,23 +33,23 @@ class ProfileData {
 
      
 
-    passRender(){
+    renderPass() {
         profile.innerHTML += 
              `<div class = "stampContainer">
                 <p class = "passStamp">NOPE</p>
               </div>`
     }
 }
 let doggy = {}
+let index = 0;
 
 function getNewDoggy(){
-    const nextDoggy = dogs[dogsNames.shift()]
+    const nextDoggy = dogs[dogsNames[index % dogsNames.length]];
+    index++;
     console.log(nextDoggy) 
-    doggy = nextDoggy ? new ProfileData(nextDoggy) : {}
-    profile.innerHTML = doggy.renderProfile()
-    console.log(dogsNames.length)
-    if(dogsNames.length === 0){
-        dogsNames = ["Rex",'Bella','Teddy']
```

Comment:
try to avoid multiple places of constant values. 
it should be somewhere totally constant and you don't need to go to multiple places to update values. 

Ps: it's more related to the team work projects, when someone just have to add\fix smth and do not know about places to update